### DISCUSSION TITLE
请问探索功能如何使用？

### DISCUSSION THREAD

**Discussion by Rookiewan** [usememos/memos#242] [Q&A] [answered]
目前使用 `0.4.4-prod` 点击探索后展示空，哪里可以看到该功能介绍？怎样才有数据？谢谢

![image](https://user-images.githubusercontent.com/13617296/191922162-96f3c06f-e3a1-417f-9846-9f100603f464.png)

**Comment by boojack** [comment, marked as answer]
可以参考一下这个 issue：https://github.com/usememos/memos/issues/126

**Reply by Rookiewan** [reply]
了解了，感谢！
原本以为是随机出现自己写过的 memo 的功能。🤣
